test/production/app-dir/unexpected-error/app/isr-unexpected-error/page.tsx

```diff
@@ -0,0 +1,13 @@
+export const revalidate = 0
+
+export default function UnexpectedErrorPage(props) {
+  // use query param to only throw error during runtime, not build time
+  if (props.searchParams.error) {
+    throw new Error('Oh no')
+  }
+  return (
+    <>
+      <p id="page">/unexpected-error</p>
+    </>
+  )
```

Comment:
```suggestion
  return <p id="page">/unexpected-error</p>
```

and same for the two other `page.tsx` files.

Also if I recall correctly, isn't ISR only for cases where `revalidate` value is a positive number? `export const revalidate = 0` [should make the page dynamically rendered](https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#revalidate) (SSR).

Reply:
Thanks! Good points. I'll update the PR